quantecon/optimize/__init__.py

```diff
@@ -3,4 +3,4 @@
 """
 
 from .scalar_maximization import brent_max
-from .root_finding import newton, newton_halley, newton_secant
+from .root_finding import *
```

Comment:
hi @spvdchachan I would rather list the items here rather than using `*`. That way if someone adds a support function that isn't prepended with `_` it won't get promoted through `import`.

Reply:
actually I just realised you have used ``__all__`` in ``root_finding.py``. Please ignore.